client/src/assets/Images/Spielplatzfinder-Dateien/bundle.js

```diff
@@ -0,0 +1,963 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
hm, what is this? This doesn't look like an image

Reply:
no idea how it appeared ❌